pom.xml

```diff
@@ -73,22 +73,27 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <jakarta.annotation.version>1.3.5</jakarta.annotation.version>
-        <jakarta.enterprise.cdi.version>2.0.1</jakarta.enterprise.cdi.version>
+        <jakarta.ejb.version>4.0.1</jakarta.ejb.version>
```

Comment:
If we agree to work on Jakata-Core, we should not include EJB on the project: 
https://jakarta.ee/release/10/images/jakarta-ee-10-platform.svg
@njr-11 what do you think?

Reply:
> If we agree to work on Jakata-Core, we should not include EJB on the project: https://jakarta.ee/release/10/images/jakarta-ee-10-platform.svg @njr-11 what do you think?

I agree, let's omit the EJB dependency given that the specification does not currently make any requirements with respect to EJBs that would need to be tested in the TCK.  I expect that at some point we will want to define how data repositories behave with respect to container managed transactions when the underlying resource is transaction capable, but that could be tested with the CDI-based `@Transactional` rather than with EJBs.

Reply:
@otaviojava @njr-11 EJBs have been removed.  Thanks for the feedback.  Just put them to make sure the project was set up correctly for the Arquillian EJB enricher.  I agree that sticking with CDI's `@Transactional` is the best move 👍🏼